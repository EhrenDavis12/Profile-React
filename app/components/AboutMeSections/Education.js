// import PropTypes from 'prop-types';
import React from 'react';
import Banner from 'components/Banners/Banner';
import { P } from 'components/SharedStyles/styles';

function Education() {
  return (
    <div className="container">
      <Banner>
        <h4>Continuing Education</h4>
        <P>JANUARY, 2019 - PRESENT</P>
        <P>Utilizing Pluralsight Online Classes</P>
        <P>
          Continuing education to maintain up to date knowledge on new
          languages, trends, and skills
        </P>
      </Banner>

      <Banner>
        <h4>Private Executive Leadership Coaching</h4>
        <P>July, 2022 - PRESENT</P>
        <P>Private Leadership coaching taken biweekly</P>
      </Banner>

      <Banner>
        <h4>Private Dance Lessons (WCS)</h4>
        <P>August, 2017 - PRESENT</P>
        <P>Private West Coast Swing Dance lessons taken weekly</P>
      </Banner>

      <Banner>
        <h4>Full Stack Web Development Certificate</h4>
        <P>JANUARY, 2019</P>
        <P>University of Arizona</P>
      </Banner>

      <Banner>
        <h4>Bachelor of Science in Game & Simulation Development</h4>
        <P>JUNE 2008 - JUNE 2011</P>
        <P>Devry University</P>
      </Banner>
    </div>
  );
}

export default Education;
