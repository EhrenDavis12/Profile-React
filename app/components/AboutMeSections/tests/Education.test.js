import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Education from '../Education';

describe('<Education />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<Education />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
