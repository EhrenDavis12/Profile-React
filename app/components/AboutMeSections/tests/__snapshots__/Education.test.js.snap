// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Education /> should render the page 1`] = `
<div
  className="container"
>
  <BannerSection>
    <h4>
      Continuing Education
    </h4>
    <styles__P>
      PRESENT
    </styles__P>
    <styles__P>
      Utilizing Pluralsight Online Classes
    </styles__P>
    <styles__P>
      Continuing education to maintain up to date knowledge on new languages, trends, and skills
    </styles__P>
  </BannerSection>
  <BannerSection>
    <h4>
      Full Stack Web Development Certificate
    </h4>
    <styles__P>
      JANUARY ,2019
    </styles__P>
    <styles__P>
      University of Arizona
    </styles__P>
  </BannerSection>
  <BannerSection>
    <h4>
      Bachelor of Science in Game & Simulation Development
    </h4>
    <styles__P>
      JUNE 2008 - JUNE 2011
    </styles__P>
    <styles__P>
      Devry University
    </styles__P>
  </BannerSection>
</div>
`;
